core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -461,19 +461,17 @@ class UtilsSuite extends SparkFunSuite with ResetSystemProperties with Logging {
     def assertResolves(before: String, after: String): Unit = {
       // This should test only single paths
       assume(before.split(",").length === 1)
-      // Repeated invocations of resolveURI should yield the same result
       def resolve(uri: String): String = Utils.resolveURI(uri).toString
+      assert(resolve(before) === after)
```

Comment:
I think the change is OK @HyukjinKwon -- we do want to check that `before` resolves to `after` and this case seems missed in this fixture. Or do I miss something? The rest of the change is just for consistency